clients/src/main/java/org/apache/kafka/common/security/JaasContext.java

```diff
@@ -54,7 +54,7 @@ public class JaasContext {
      * If the property is not defined, the context will contain the default Configuration and
      * the context name will be one of:
      * <ol>
-     *   <li>Lowercased listener name followed by a period and the string `KafkaServer`</li>
+     *   <li>Lowercase listener name followed by a period and the string `KafkaServer`</li>
```

Comment:
I think this change is unnecessary since there are many similar usage of non-standard adjective, for example, "Revertable" and "Interruptible", right?

Reply:
I see, thx!